tests/playwright/src/specs/rh-extension-pack-installation.spec.ts

```diff
@@ -0,0 +1,154 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Page } from '@playwright/test';
+import { expect as playExpect, test } from '@playwright/test';
+
+import { extensionsPackExtension, extensionsRHPackList } from '../model/core/extensions';
+import { ExtensionCatalogCardPage } from '../model/pages/extension-catalog-card-page';
+import { ExtensionsPage } from '../model/pages/extensions-page';
+import { WelcomePage } from '../model/pages/welcome-page';
+import { NavigationBar } from '../model/workbench/navigation';
+import { Runner } from '../runner/podman-desktop-runner';
+
+const DISABLED = 'DISABLED';
```

Comment:
I think that these constants can be extracted into `model/core/states.ts` module. They can be reused across tests.

Reply:
please open a follow up issue to switch to using the constants/enums for the states from there in other tests as well.